pages/system/radios.js

```diff
@@ -92,46 +107,44 @@ export default class SystemPageRadios extends React.Component {
     ),
   }
 ];`}
-        </System.CodeBlock>
-        <br />
-        <System.P>
-          Define the default selected option and handle the state changes when the users selects a different option.
-        </System.P>
-        <br />
-        <System.CodeBlock>
-          {`state = {
-  default: '2',
-};
-
-_handleChange = (e) => {
-  this.setState({ [e.target.name]: e.target.value });
-};`}
         </System.CodeBlock>
         <br />
         <System.P>Declare the RadioGroup component.</System.P>
         <br />
         <System.CodeBlock>
-          {`<System.RadioGroup
-  name="five"
-  options={RADIO_GROUP_OPTIONS}
-  selected={this.state.default}
-  onChange={this._handleChange}
-/>`}
+          {`class ExampleOne extends React.Component {
+   state = { ExampleOne: '2' }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`